.github/renovate.json

```
@@ -4,6 +4,7 @@
   "schedule": ["at any time"],
   "semanticCommits": "enabled",
   "reviewers": ["@Jolg42", "@millsp", "@aqrln", "@SevInf", "@danstarns", "@jkomyno"],
+  "schedule": ["after 9pm, before 8:00am"],
```

```suggestion
  "schedule": [
    "after 9pm every weekday",
    "before 8am every weekday",
    "every weekend"
  ]
```

Not sure if `every weekday` is actually needed.